public/javascript/components/surveyController.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Survey Controller
+ */
+
+import $ from 'jquery';
+import firebase from "firebase/app";
+import "firebase/database";
+
+// Submit button on click
+$('.survey__submit').click(function(e) {
+  let getCheckedButtons = getAllRadioButtonsChecked()
+  
+  e.preventDefault()
+
+  if (checkAllButtonsClicked()) {
+    writeDataIntoFirebase(getCheckedButtons)
+  }
+})
+
+// Validation to see if buttons are all checked
+const checkAllButtonsClicked = () => {
+  let checkState = true
+
+  $('.survey__question').each(function() {
+    if (!$(this).find('input:radio').is(':checked')) {
+      checkState = false
+    }
+  })
+
+  if(checkState) 
+    alert('all buttons checked')
```

Comment:
Do we need to merge these alerts into your develop branch? 😂

Reply:
Yes for sure!